.github/ubiquibot-config.yml

```
@@ -28,6 +28,15 @@ priority-labels:
     weight: 4
   - name: "Priority: 4 (Emergency)"
     weight: 5
+default-labels:
+  global:
+    - "Time: <1 Hour"
+    - "Priority: 0 (Normal)"
+    - "Test"
+  users:
+    pavlovcik:
+      - "Time: <1 Hour"
+      - "Priority: 0 (Normal)"
```

There's one thing I forgot to highlight @whilefoo but https://github.com/ubiquity/ubiquibot/issues/500#issuecomment-1634429276 

Would you mind refactoring this config to be like this:

```suggestion
default-labels:
  - "Time: <1 Hour"
  - "Priority: 0 (Normal)"
  - "Test"
```